torch/autograd/profiler.py

```diff
@@ -114,6 +120,12 @@ def table(self, sort_by=None, row_limit=100, header=None):
                 Valid keys include: ``cpu_time``, ``cuda_time``, ``cpu_time_total``,
                 ``cuda_time_total``, ``cpu_memory_usage``, ``cuda_memory_usage``,
                 ``self_cpu_memory_usage``, ``self_cuda_memory_usage``, ``count``.
+            top_level_events_only(bool, optional): Boolean flag to determine the
+                selection of events to display. If true, the profiler will only
+                display events at top level like top-level invocation of python
+                `lstm`, python `add` or other functions, nested events like cpu/cuda
+                ops events like `EmbeddingBagBackward`, `_embedding_bag_backward` are
```

Comment:
I would remove explicit examples here, and also it's not clear why e.g. EmbeddingBagBackward should not be printed